tpm2/tpm2.go

```diff
@@ -1594,6 +1614,24 @@ func Certify(rw io.ReadWriter, parentAuth, ownerAuth string, object, signer tpmu
 	return decodeCertify(resp)
 }
 
+// CertifyEx generates a signature of a loaded TPM object with a signing key
+// signer. This function differs from Certify in that it takes the scheme
```

Comment:
Can you also add a comment to `Certify` that explains it will hardcode `RSAPSS` and `SHA256` for the signing scheme?

Reply:
Oops, I miswrote. Thank you for correctly using `RSASSA` in the actual change 😃

Reply:
No worries, I figured it was just a typo ;).